### DISCUSSION TITLE
Issue with Kibana after upgrading security onion to vs 2.3.100

### DISCUSSION THREAD

**Discussion by pavlos1982** [Security-Onion-Solutions/securityonion#7130] [General] [answered]
Hi All,

I am getting a notification that keeps popping up with the new security onion Kibana service that shows the following message " The data you are seeing might be incomplete or wrong", please see the screenshot, thanks
![1](https://user-images.githubusercontent.com/5363713/152759475-5f3be554-21e9-417d-ad75-c25bef16bef6.JPG)
![2](https://user-images.githubusercontent.com/5363713/152759521-97bc8371-8ab5-4fae-9af5-d846327bf608.JPG)

**Comment by dougburks** [comment, marked as answer]
Have you tried the suggestions in the `soup` documentation?

From https://docs.securityonion.net/en/2.3/soup.html#kibana:
> If Kibana loads but the dashboards display errors that they didn’t before the upgrade, first shift-reload your browser to make sure there are no cache issues. If that doesn’t resolve the issue, then you may need to reload the dashboards on your manager:
```
sudo salt-call state.apply kibana.so_savedobjects_defaults -l info
```

**Reply by pavlos1982** [reply]
Thanks, seem to be working now, will monitor and see how it goes.
